src/components/button/ButtonConstants.ts

```diff
@@ -22,3 +24,8 @@ export const MIN_WIDTH = {
 };
 
 export const DEFAULT_SIZE = ButtonSize.large;
+
+export const ICON_SOURCE_MARGIN = {
```

Comment:
This `ICON_SOURCE_MARGIN` constant is for SVG image usage, instead of having this constant in the `ButtonConstants` file can we add this functionality to the `asCss` function in the `svgImage` index file?

In the `asCss` we will change the margin from `8` to `8px` instead of dealing with it inside the `ButtonConstants` file.